ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.1.yaml

```diff
@@ -14,19 +14,24 @@ base_images:
   tests-private-postupg:
     name: tests-private
     namespace: ci
-    tag: "4.16"
+    tag: "4.17"
   tests-private-preupg:
     name: tests-private
     namespace: ci
-    tag: "4.15"
+    tag: "4.16"
   tools:
-    name: "4.16"
+    name: "4.17"
     namespace: ocp
     tag: tools
   verification-tests:
     name: verification-tests
     namespace: ci
     tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```

Comment:
Is the `build_root` needed ?

Reply:
Hmm, I'm not familiar with the image. We just wanted to ensure the image configuration is consistent across all versions. Is there any side effect if we add an image but not required?

Reply:
In the earlier days, I copies those configs from dev.  And later we are more familiar with them, we opt out some of the configs because we know they are not needed/used.

Reply:
Generally in most of the situations, if we know it works already, we do not add extra things.

Reply:
In 4.16 chain upgrade, some config files contain it but some don't. So do you suggest we drop it from all config files? cc @jianlinliu for discussion

Reply:
I'm prefer not adding them here, and try to remove the others in a separate PR.

Reply:
I have no idea about the purpose of the image, but I agree with that, if the ci jobs do not need the image, then we should drop it.

Reply:
okay, how about we drop it from all files in a separate PR? The PR has been open for a long while and I was hoping it can be merged soon.

Reply:
so the decision is that remove build_root from all chain upgrade configs?

Reply:
okay for me.

Reply:
I'm prefer not adding them here, and try to remove the others in a separate PR.

Reply:
Adding it here and then use another PR to remove it does not looks like a good process.

Reply:
I understand your concern Liang. But the code change in the PR is too big. We want to avoid reworking the PR repeatedly unless absolutely necessary. Does it make sense to you?

Reply:
done